src/store/index.js

```diff
@@ -25,10 +25,7 @@ export default new Vuex.Store({
   state: {
     isRestored: false,
     mnemonic: null,
-    current: {
-      network: NETWORK_DEFAULT.name,
-      token: 0,
-    },
+    network: NETWORK_DEFAULT.name,
```

Comment:
I think this is a good change, but I'm not sure if adding new migrations now is a good thing. Especially as I'm working on removing `network` from the store within this task: https://aeternity-blockchain.atlassian.net/browse/SW-822